rust-runtime/aws-smithy-types/src/macros.rs

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Various utility macros to aid runtime crate writers.
```

Comment:
Should this be in `aws-smithy-runtime-api` instead?

Reply:
I'm not sure, as we still use the `types` crate for stuff and the macro isn't orchestrator specific.

I'll move it.